### PR TITLE
Candy monitoring stack

### PR DESCRIPTION
**Pull request by PatStLouis**
- Syntax correction in manage script
- Fix api path url in telegraph config file
- Configuration changes to monitor Candy-dev network

### CODE REVIEW

File: telegraf/telegraf.conf [hunk 1]
```diff
@@ -14,7 +14,7 @@
   omit_hostname = false
 
 [[outputs.influxdb]]
-  urls = ["http://influxdb:8086"]
+  urls = ["http://influxdb:8086/api/v2"]
```
Comment (PatStLouis) [inline telegraf/telegraf.conf:17]:
```suggestion
  urls = ["http://influxdb:8086"]
```

File: manage [hunk 2]
```diff
@@ -480,7 +481,7 @@ if ! isInstalled ${DOCKER_EXE}; then
   exit 1
 fi
 
-DOCKER_COMPOSE_EXE="${DOCKER_EXE} compose"
+DOCKER_COMPOSE_EXE="${DOCKER_EXE}-compose"
```
Comment (WadeBarnes) [inline manage:484]: This change should be reverted.  `docker-compose` is deprecated.

File: telegraf/telegraf.conf (hunk 1 above)
Comment (WadeBarnes) [inline telegraf/telegraf.conf:17]: This seems to be incompatible with InfluxDB v1.8.  It breaks all of the existing controls trying to access the InfluxDB data sources.  This should be combined with a complete project update to InfluxDB v2.x.

File: telegraf/telegraf.conf [hunk 3]
```diff
@@ -67,7 +67,7 @@
   ## Export metric collection time.
   # export_timestamp = false
 
-[[inputs.http]]
+  [[inputs.http]]
```
Comment (WadeBarnes) [inline telegraf/telegraf.conf:70]: Remove leading whitespace.

File: fetch-validator-status/networks.json [hunk 4]
```diff
@@ -50,5 +50,9 @@
   "itn": {
     "name": "Indicio TestNet",
     "genesisUrl": "https://raw.githubusercontent.com/Indicio-tech/indicio-network/main/genesis_files/pool_transactions_testnet_genesis"
+  },
```
Comment (WadeBarnes) [inline fetch-validator-status/networks.json:53]: The `main` branch has updated the network list.  I'll be rebasing the `monitoring-stack` branch on it.

File: telegraf/telegraf.conf [hunk 5]
```diff
@@ -507,4 +507,151 @@
   ##   1. Local               -- interpret based on machine localtime
   ##   2. "America/New_York"  -- Unix TZ values like those found in https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
   ##   3. UTC                 -- or blank/unspecified, will return timestamp in UTC
-  json_timezone = ""
\ No newline at end of file
+  json_timezone = ""
+
+[[inputs.http]]
```
Comment (WadeBarnes) [inline telegraf/telegraf.conf:512]: This is fine.  The network code for the CANdy-Dev network will still be `cdn`.

**Review by WadeBarnes — CHANGES_REQUESTED** [review]
Some small changes to make.

**Comment by WadeBarnes** [comment]
@PatStLouis, I've rebased the `monitoring-stack` branch on the latest from the Hyperledger `main` branch.  You should rebase your PR on the updates, and make the changes I've suggested above.  Then I can merge the changes and get it all merged into the Hyperledger  `monitoring-stack` branch.

**Comment by PatStLouis** [comment]
Thanks for the thorough review @WadeBarnes I will take actions and get back to you when the PR is ready.

**Pull request closed without merging**